Windows builds using the automatic include directory configuration fail

```
D:\a\nobind\nobind\example\hello.cc(33,10): fatal error C1083: Cannot open include file: 'nobind.h': No such file or directory [D:\a\nobind\nobind\example\build\hello.vcxproj]
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [1.1.1]
+
+ -  Fix [#7](https://github.com/mmomtchev/nobind/issues/7), escape the include directory path on Windows
+
 ## [1.1.0] 2023-11-30
 
 -   Implement [#5](https://github.com/mmomtchev/nobind/issues/5), automatic object persistence in asynchronous mode

--- a/index.js
+++ b/index.js
@@ -1,5 +1,5 @@
 const path = require('path');
 
 module.exports = {
-  include: path.resolve(__dirname, 'include')
+  include: `"${path.resolve(__dirname, 'include')}"`
 };

